PRIVACY.md

```diff
@@ -38,41 +38,42 @@ These third parties may log additional information besides your IP address and t
+| [Directory of Open Access Books](https://www.doabooks.org/)                                                                  | <https://www.doabooks.org/en/resources/accessibility> |
+| [Digitala Vetenskapliga Arkivet](https://www.diva-portal.org/)                                                               | <https://www.uu.se/en/about-uu/data-protection-policy/> |
+| [DOI Foundation](https://www.doi.org/)                                                                                       | <https://www.doi.org/privacy-policy/> |
+| [Elsevier](https://www.elsevier.com/)                                                                                        | <https://www.elsevier.com/legal/privacy-policy> |
+| [Google Scholar](https://scholar.google.com/)                                                                                | <https://policies.google.com/privacy> |
+| [Gemeinsamer Verbundkatalog](https://www.gbv.de/)                                                                            | <https://www.gbv.de/datenschutz> |
+| [IACR](https://www.iacr.org/)                                                                                                | <https://www.iacr.org/privacy.html> |
+| [IEEEXplore](https://ieeexplore.ieee.org/Xplore/home.jsp)                                                                    | <https://www.ieee.org/security-privacy.html> |
+| [INSPIRE](https://inspirehep.net/)                                                                                           | <https://cern.service-now.com/service-portal?id=privacy_policy&se=INSPIRE-Online&notice=main> |
+| [JSTOR](https://www.jstor.org/)                                                                                              | <https://www.ithaka.org/privacypolicy/> |
+| [Library of Congress](https://lccn.loc.gov/)                                                                                 | <https://www.loc.gov/legal/> |
+| [National Library of Medicine](https://www.ncbi.nlm.nih.gov/)                                                                | <https://www.nlm.nih.gov/web_policies.html> |
+| [MathSciNet](http://www.ams.org/mathscinet)                                                                                  | <https://www.ams.org/about-us/privacy> |
+| [mEDRA](https://www.medra.org/)                                                                                              | <https://<https://www.medra.org/stdoc/en/Servizio_DOI_Informativa_ENG.pdf> |
+| [Mr. DLib](https://mr-dlib.org/) [1]                                                                                         | <https://support.dataverse.harvard.edu/harvard-dataverse-privacy-policy> |
+| [OpenAI](https://openai.com/) [1]                                                                                           | <https://openai.com/policies/privacy-policy/> |
```

Comment:
The | is off by one character.

You can use http://markdowntable.com/

Please file a separate PR for this Table.

Reason: We will get this change in earlier and the other changes. Moreover, this change is independent form the otehr changes.